php/php.api.phpmodule/src/org/netbeans/modules/php/api/PhpVersion.java

```diff
@@ -250,6 +250,17 @@ public boolean hasNeverType() {
         return this.compareTo(PhpVersion.PHP_81) >= 0;
     }
 
+    /**
+     * Check whether this version supports final const. (as of PHP 8.1)
+     *
+     * @return {@code true} if this version supports final const, {@code false}
+     * otherwise
+     * @since 2.101
+     */
+    public boolean hasFinalConst() {
```

Comment:
We don't need `hasFinalMethod()`?

Reply:
We can use a `final` method as of PHP 5.0. So, it doesn't seem necessary :)

Reply:
Stupid me :sweat_smile: